doc/nrf/protocols/matter/overview/data_model.rst

```diff
@@ -434,6 +451,106 @@ Energy device types
 | Heat Pump          | A Heat Pump device is a device that uses electrical energy to heat either    | Certifiable         |                                       |
 |                    | spaces or water tanks using ground, water or air as the heat source.         |                     |                                       |
 +--------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Meter Reference    | A Meter Reference Point device provides details about tariffs and metering.  | Certifiable         |                                       |
+| Point              |                                                                              |                     |                                       |
++--------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Electrical Energy  | A Electrical Energy Tariff is a device that defines a tariff                 | Certifiable         |                                       |
+| Tariff             | for the consumption or production of electrical energy.                      |                     |                                       |
```

Comment:
Shouldn't we just write the whole name in one line, and expand the table size? This is the point for all entries here.

Reply:
I don't know, I have just followed the existing pattern for all entries and a lot of them is written in multiple lines